src/windshaft/filters/category.js

```diff
@@ -140,5 +140,10 @@ module.exports = WindshaftFilterBase.extend({
 
   getRejectedCategoryNames: function () {
     return this.rejectedCategories.map(function (category) { return category.get('name'); });
+  },
+
+  remove: function () {
+    this.cleanFilter();
```

Comment:
From what I see we can decide here if `cleanFilter` should or should not trigger a `change` event (it admits a boolean param)... Do we want it to trigger this event? Does this have any side effect? (I assume not but just in case...)